### DISCUSSION TITLE
z/OS Open Source APAC Guild Meeting #1 - 30th March, 2023 - 9.00 AM IST

### DISCUSSION THREAD

**Discussion by HarithaIBM** [zopencommunity/meta#252] [Guild]
**Agenda:**

- Motivation behind z/OS Open Source Software (OSS) Initiative 
- Goals for the OSS Guild 
- A review of updates and progress 
- Highlights of new zopen features
- Live demonstrations of:
- Latest zopen enhancements
- Polling and feedback
- Q/A

**How to Join?**
Join link: https://ibm.webex.com/ibm/j.php?MTID=m37be3c328a6075798b1fe5df282eac2f
Webinar number: 145 181 3705 
Webinar password: htMm5vSWU52 (48665879 from phones)

**Date and Time**
Thursday, March 30, 2023
9:00 AM  |  (UTC+05:30) Chennai, Kolkata, Mumbai, New Delhi  |  1 hr

**Recording/Slide/Poll/Chat Log**

- [APAC_ZOS_OSS_Guild#1_Mar.pptx](https://github.com/ZOSOpenTools/meta/files/11118494/APAC_ZOS_OSS_Guild.1_Mar.pptx)
- [chat_APAC_Guild#1_Mar.csv](https://github.com/ZOSOpenTools/meta/files/11118499/chat_APAC_Guild.1_Mar.csv)
- [poll-results_APAC_Guild#1_Mar..pdf](https://github.com/ZOSOpenTools/meta/files/11118512/poll-results_APAC_Guild.1_Mar.pdf)
- [Guild Recording](https://ibm.ent.box.com/folder/201374290632)

**Comment by MikeFultonDev** [comment]
Looking forward to it! Can you update the title to include the date and time so people can plan?

**Reply by HarithaIBM** [reply]
> Looking forward to it! Can you update the title to include the date and time so people can plan?
I have updated it, Mike!

**Comment by HarithaIBM** [comment]
[invite.ics.zip](https://github.com/ZOSOpenTools/meta/files/11090289/invite.ics.zip)

**Reply by lbdyck** [reply]
nice

**Comment by daver1958** [comment]
Hi,
I am getting the box error message "Oops! We can't seem to find the page you're looking for." when i try and access the Guild Recording url above.
Regards Dave

**Reply by HarithaIBM** [reply]
> Hi, I am getting the box error message "Oops! We can't seem to find the page you're looking for." when i try and access the Guild Recording url above. Regards Dave

May i know your email id , please!

**Reply by HarithaIBM** [reply]
Can you please try accessing now 
https://ibm.box.com/s/myq3um89i0qmqzkqjewfxyjhjmnnvqmu

**Reply by daver1958** [reply]
Works now. Thanks

**Comment by MikeFultonDev** [comment]
@IgorTodorovskiIBM fya

**Comment by MikeFultonDev** [comment]
@HarithaIBM fya - make sure it didn’t expire. The default expiry is very fast with box

**Reply by HarithaIBM** [reply]
I will check this Mike, Thank you!
